### PR TITLE
updates news for MASVS 1.2

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide project website. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

File: tab_news.md [hunk 1]
```diff
@@ -8,9 +8,23 @@ tags: mstg
...
+- French
+- German
+- Japanese
+- Korean
+- Russian
+- Spanish
+
+The MASVS and its translations are availabe in PDF, Mobile, ePub, docx and you can also read it via Gitbook. See here for details: <https://github.com/OWASP/owasp-masvs#getting-the-masvs>
+
+The project team (Sven Schleier, Jeroen Willemsen and Carlos Holguera)  would like to thank all the contributors, translators and those who build the improved automation around it and all their hard work and support in the last few months!  New releases will be much faster thanks to our GitHub actions and Docker containers. Next to that, we are happy to add Korean and Chinese Simplified to our ever growing list of translations! We will finalize the document generation system and then apply the same build system to the Mobile Security Testing Guide (MSTG) in order to speed up the release process and release more frequently.
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
The project team (Sven Schleier, Jeroen Willemsen and Carlos Holguera) would like to thank all the contributors, translators and those who build the improved automation around it and all their hard work and support in the last few months! New releases will be much faster thanks to our GitHub actions and Docker containers. Next to that, we are happy to add Korean and Chinese Simplified to our ever growing list of translations! We will finalize the document generation system and then apply the same build system to the Mobile Security Testing Guide (MSTG) in order to speed up the release process and release more frequently.
```

File: tab_news.md [hunk 2]
```diff
@@ -8,9 +8,23 @@ tags: mstg
...
-New releases will be much faster thanks to our GitHub actions and docker containers. Next to that, we are happy to add Korean and Chinese Simplified to our ever growing list of translations!
-We will finalize the document generation system and then apply the same build system to the MSTG in order to speed up the release process and release more frequently.
+A new version of the OWASP Mobile Application Security Standard (MASVS) was released! The MASVS establishes baseline security requirements for mobile apps and summarizes them in one standard. With this new release we achieved a significant alignment and coverage with existing mobile security documents from ENISA, older NIST documents, OWASP Mobile top 10, and others. The new version 1.2 is available in Github Releases: <https://github.com/OWASP/owasp-masvs/releases/tag/v1.2>. For more details please look into our Changelog for Version 1.2 and Version 1.2-RC <https://github.com/OWASP/owasp-masvs/blob/master/Document/CHANGELOG.md>.
+
+Due to the great support of our community we have now 9 different languages available in total for the MASVS:
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
Thanks to the great support of our community we have now 9 different languages available in total for the MASVS:
```

File: tab_news.md (hunk 1 above)
Comment (commjoen) [inline tab_news.md]:
```suggestion
The project team (Sven Schleier, Jeroen Willemsen and Carlos Holguera) would like to thank all the contributors, translators and those who build the improved automation around it and all their hard work and support in the last few months!  New releases will be much faster thanks to our GitHub actions and Docker containers. Next to that, we are happy to add Korean and Chinese Simplified to our ever growing list of translations! We will finalize the document generation system and then apply the same build system to the Mobile Security Testing Guide (MSTG) in order to speed up the release process and release more frequently.
```
And maybe: should we add the translators that have been active for us?

File: tab_news.md [hunk 3]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
+- French - Romuald Szkudlarek, Abderrahmane Aftahi, Christian Dong (Review)
+- German - Rocco Gränitz, and Sven Schleier
+- Japanese - Koki Takeyama, Riotaro Okada (Review)
+- Korean - Youngjae Jeon, Jeongwon Cho, Jiyou Han, and Jiyeon Sung
+- Russian - Eugen Martynov, Gall Maxim, Chelnokov Vladislav (Review), Oprya Egor (Review), Tereshin Dmitry (Review) |
+- Spanish - Martin Marsicano, and Carlos Holguera
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- Spanish - Martin Marsicano and Carlos Holguera
```

File: tab_news.md [hunk 4]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
+- French - Romuald Szkudlarek, Abderrahmane Aftahi, Christian Dong (Review)
+- German - Rocco Gränitz, and Sven Schleier
+- Japanese - Koki Takeyama, Riotaro Okada (Review)
+- Korean - Youngjae Jeon, Jeongwon Cho, Jiyou Han, and Jiyeon Sung
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- Korean - Youngjae Jeon, Jeongwon Cho, Jiyou Han and Jiyeon Sung
```

File: tab_news.md [hunk 5]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
+- French - Romuald Szkudlarek, Abderrahmane Aftahi, Christian Dong (Review)
+- German - Rocco Gränitz, and Sven Schleier
+- Japanese - Koki Takeyama, Riotaro Okada (Review)
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- Japanese - Koki Takeyama and Riotaro Okada (Review)
```

File: tab_news.md [hunk 6]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- Korean
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
+- French - Romuald Szkudlarek, Abderrahmane Aftahi, Christian Dong (Review)
+- German - Rocco Gränitz, and Sven Schleier
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- German - Rocco Gränitz and Sven Schleier
```

File: tab_news.md [hunk 7]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- Japanese
-- Korean
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
+- French - Romuald Szkudlarek, Abderrahmane Aftahi, Christian Dong (Review)
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- French - Romuald Szkudlarek, Abderrahmane Aftahi and Christian Dong (Review)
```

File: tab_news.md [hunk 8]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- German
-- Japanese
-- Korean
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
+- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera, and Jeroen Beckers
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- English - Jeroen Willemsen, Sven Schleier, Carlos Holguera and Jeroen Beckers
```

File: tab_news.md [hunk 9]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- French
-- German
-- Japanese
-- Korean
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
+- Chinese (simplified) - Bob Peng, Harold Zang, and Jack S
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- Chinese (simplified) - Bob Peng, Harold Zang and Jack S
```

File: tab_news.md [hunk 10]
```diff
@@ -10,21 +10,21 @@ tags: mstg
...
-- English
-- French
-- German
-- Japanese
-- Korean
-- Russian
-- Spanish
+Due to the great support of our community we have now 9 different languages available in total for the MASVS and would like to thank all of our translators for their great work and support throughout:
+
+- Chinese (traditional) - Peter Chi, and Lex Chien, Henry Hu, Leo Wang
```
Comment (cpholguera) [inline tab_news.md]:
```suggestion
- Chinese (traditional) - Peter Chi, Lex Chien, Henry Hu and Leo Wang
```

**Pull request merged**